### PR TITLE
Small improvements: Bump CryptoSwift, bubble up errors and delete the unneeded .pbxproj

### PR DESCRIPTION
**Pull request by Lascorbe**
Prob needs a v1.8.0 release, it introduces a new delegate method that must be implemented

### CODE REVIEW

**Review by CassiusPacheco — COMMENTED** [review]
I can't approve it, but LGTM. I'd love to see a new version out. There were already other things merged in that haven't been push out as a new version yet.

File: Sources/PublicInterface/Server.swift [hunk 1]
```diff
@@ -26,6 +26,9 @@ public protocol ServerDelegate: AnyObject {
 
     /// Called only when the session is updated with intention of the dAppt.
     func server(_ server: Server, didUpdate session: Session)
+    
+    /// Called when not able to parse a message
+    func server(_ server: Server, couldNotParseMessage message: String, from url: WCURL, error: Error)
```
Comment (DmitryBespalov) [inline Sources/PublicInterface/Server.swift:31]: Could you please move this method to `ServerDelegateV2` protocol? This way, this doesn't break existing code

File: Sources/PublicInterface/Server.swift [hunk 2]
```diff
@@ -101,13 +103,19 @@ open class Server: WalletConnect {
...
+            try handle(request)
         } catch {
             LogService.shared.error(
                 "WC: incomming text deserialization to JSONRPC 2.0 requests error: \(error.localizedDescription)"
             )
-            // TODO: handle error
-            try! send(Response(url: url, error: .invalidJSON))
+            delegate?.server(self, couldNotParseMessage: text, from: url, error: error)
+            do {
+                try send(Response(url: url, error: .invalidJSON))
```
Comment (DmitryBespalov) [inline Sources/PublicInterface/Server.swift:113]: 👍

File: Sources/PublicInterface/Server.swift (hunk 1 above)
Comment (Lascorbe) [reply Sources/PublicInterface/Server.swift:31]: if you implemented `ServerDelegateV2` you will run into the same issue, should I create a `ServerDelegateV3`?

Comment (Lascorbe) [reply Sources/PublicInterface/Server.swift:31]: @DmitryBespalov this is probably going to be unmaintained soon, do you still want this PR to be merged?

**Pull request closed without merging**